parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -353,8 +353,7 @@ try
                 -KubeClusterCIDR $global:KubeClusterCIDR `
                 -MasterSubnet $global:MasterSubnet `
                 -KubeServiceCIDR $global:KubeServiceCIDR `
-                -VNetCIDR $global:VNetCIDR `
-                -TargetEnvironment $TargetEnvironment
+                -VNetCIDR $global:VNetCIDR -IsAzureStack {{if IsAzureStackCloud}}$true{{else}}$false{{end}}

```

Comment:
Can you move the if/else to a variable assignment before the Set-AzureCNICall and all add a comment explaining what is actually going on?

Reply:
and revert back to one param per line (for consistency?)